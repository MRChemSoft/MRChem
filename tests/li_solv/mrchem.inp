# vim:syntax=sh:
world_prec = 1.0e-3               # Overall relative precision
world_size = 5                    # Size of simulation box 2^n

Molecule {
  charge = 1
$coords
Li   0.0     0.0    0.0
$end
}

WaveFunction {
  method = pbe0                     # Wave function method (HF or DFT)
  restricted = true
}

Properties {
  reaction_energy = true            # Compute ground state energy
}

Cavity {
  $coords
  0.0     0.0    0.0  4.0
  $end
  width = 0.5
  epsilon_out = 2.0
}

SolventEffect{
  kain = 5
}

SCF {
  run = true
  kain = 5                        # Length of KAIN iterative history
  orbital_thrs = 1.0e-3           # Convergence threshold in orbital residual
  guess_type = mw                 # Type of initial guess: none, mw, gto
}