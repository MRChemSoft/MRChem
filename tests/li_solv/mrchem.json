{
  "mpi": {
    "numerically_exact": true,
    "shared_memory_size": 10000
  },
  "printer": {
    "print_level": 0,
    "print_input": false,
    "print_mpi": false,
    "file_name": "mrchem",
    "print_prec": 6,
    "print_width": 70
  },
  "molecule": {
    "charge": 1,
    "multiplicity": 1,
    "coords": [
      {
        "atom": "li",
        "xyz": [
          0.0,
          0.0,
          0.0
        ]
      }
    ],
    "cavity_coords": [
      {
        "center": [
          0.0,
          0.0,
          0.0
        ],
        "radius": 4.0
      },
      {
        "center": [
          0.0,
          0.0,
          0.0
        ],
        "radius": 4.0
      }
    ],
    "cav_width": 0.5,
    "cav_AA": false
  },
  "mra": {
    "basis_type": "i",
    "order": 8,
    "boxes": [
      2,
      2,
      2
    ],
    "corner": [
      -1,
      -1,
      -1
    ],
    "min_scale": -4,
    "max_scale": 20,
    "scaling_factor": [
      1.0,
      1.0,
      1.0
    ]
  },
  "initial_guess": {
    "restricted": true,
    "guess_prec": 1e-05,
    "method": "sad",
    "zeta": 2,
    "start_orbitals": "orbitals/phi",
    "final_orbitals": "orbitals/phi",
    "file_basis": "initial_guess/mrchem.bas",
    "file_moa": "initial_guess/mrchem.moa",
    "file_mob": "initial_guess/mrchem.mob",
    "write_orbitals": false
  },
  "scf_calculation": {
    "final_orbitals": "orbitals/phi",
    "write_orbitals": false,
    "cube_plot": {
      "density": false,
      "orbital": [],
      "plotter": {
        "points": [
          20,
          20,
          20
        ],
        "O": [
          0.0,
          0.0,
          0.0
        ],
        "A": [
          1.0,
          0.0,
          0.0
        ],
        "B": [
          0.0,
          1.0,
          0.0
        ],
        "C": [
          0.0,
          0.0,
          1.0
        ]
      }
    },
    "initial_energy": {
      "method_name": "Hartree-Fock",
      "localize": false,
      "prec": 1e-05
    },
    "orbital_solver": {
      "method_name": "Hartree-Fock",
      "kain": 4,
      "max_iter": 5,
      "rotation": 0,
      "localize": false,
      "start_prec": 1e-05,
      "final_prec": 1e-05,
      "orbital_thrs": 0.01,
      "property_thrs": -1.0,
      "helmholtz_prec": 1e-05
    },
    "fock_operator": {
      "kinetic_operator": {
        "derivative": "abgv_55"
      },
      "nuclear_operator": {
        "proj_prec": 1e-05,
        "smooth_prec": 1e-05,
        "shared_memory": true
      },
      "coulomb_operator": {
        "poisson_prec": 1e-05,
        "shared_memory": true
      },
      "exchange_operator": {
        "screen": false,
        "poisson_prec": 1e-05
      },
      "reaction_operator": {
        "poisson_prec": 1e-05,
        "reaction_history": 4,
        "eps_in": 1.0,
        "eps_out": 2.0,
        "cav_lin": false
      }
    }
  },
  "rsp_calculations": []
}