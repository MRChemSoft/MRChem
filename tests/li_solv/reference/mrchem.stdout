                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                                                                ***
***        __  __ ____   ____ _                                    ***
***       |  \/  |  _ \ / ___| |__   ___ _ __ ___                  ***
***       | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                 ***
***       | |  | |  _ <| |___| | | |  __/ | | | | |                ***
***       |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                ***
***                                                                ***
***       VERSION            0.3.0-alpha                           ***
***                                                                ***
***       Git branch         new_solvent                           ***
***       Git commit hash    5789e0a6e118cd9a50c2                  ***
***       Git commit author  Gabriel                               ***
***       Git commit date    Tue Apr 21 17:28:00 2020 +0200        ***
***                                                                ***
***       Magnar Bjorgve     <magnar.bjorgve@uit.no>               ***
***       Roberto Di Remigio <roberto.di.remigio@uit.no>           ***
***       Luca Frediani      <luca.frediani@uit.no>                ***
***       Gabriel Gerez      <gsa017@post.uit.no>                  ***
***       Stig Rune Jensen   <stig.r.jensen@uit.no>                ***
***       Peter Wind         <peter.wind@uit.no>                   ***
***                                                                ***
**********************************************************************

----------------------------------------------------------------------

 MPI processes         :      (no bank)                             1
 OpenMP threads        :                                            8
 Total cores           :                                            8
                                                                      
----------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

----------------------------------------------------------------------

 MRCPP version         : 1.2.0-alpha3
 Git branch            : new_solvent
 Git commit hash       : 3f287c5097d6819ab1bc
 Git commit author     : Gabriel
 Git commit date       : Thu Apr 2 11:43:14 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP

----------------------------------------------------------------------


                                                                      
======================================================================
                       MultiResolution Analysis
----------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
----------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                     Initializing Molecule                      ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            1
 Multiplicity          :                                            1
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Ground State Wavefunction               ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial orbitals
 Method                : Project MW molecular orbitals
 Precision             : 1.00000e-03
 Restricted            : True
 MO file               : initial_guess/phi_p_scf
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            1
 Beta electrons        :                                            1
 Total electrons       :                                            2
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    2    p        :                           1.000000000000e+00
======================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial energy
 Method                : DFT (PBE0)
 Precision             : 1.00000e-03
 Localization          : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                             Running SCRF
----------------------------------------------------------------------
Iter:	
1
update:	
3.471017e+00
Iter:	
2
update:	
7.998382e-01
Iter:	
3
update:	
1.376950e-01
Iter:	
4
update:	
4.301559e-03
Iter:	
5
update:	
2.302888e-05
======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.223209232878
 E-N energy            :           (au)              -16.101226552149
 Coulomb energy        :           (au)                3.286694623873
 Exchange energy       :           (au)               -0.410827570767
 X-C energy            :           (au)               -1.259248609197
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
 Tot. Reac. Energy     :           (au)               -0.060550365924
 El. Reac. Energy      :           (au)                0.121121255093
 Nuc. Reac. Energy     :           (au)               -0.181671621017
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.140277620268
 Nuclear energy        :           (au)               -0.181671621017
----------------------------------------------------------------------
 Total energy          :           (au)           -7.321949241286e+00
                       :     (kcal/mol)           -4.594592480444e+03
                       :       (kJ/mol)           -1.922377495066e+04
                       :           (eV)           -1.992403801212e+02
======================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Optimize ground state orbitals
 Method                : DFT (PBE0)
 Checkpointing         : Off
 Max iterations        : 100
 KAIN solver           : 5
 Localization          : Off
 Diagonalization       : First two iterations
 Start precision       : 1.00000e-03
 Final precision       : 1.00000e-03
 Helmholtz precision   : 1.00000e-03
 Energy threshold      : Off
 Orbital threshold     : 1.00000e-03
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
 Iter         MO residual            Total energy              Update
----------------------------------------------------------------------
    0        1.000000e+00         -7.321949241286       -7.321949e+00
======================================================================
                             Running SCRF
----------------------------------------------------------------------
Iter:	
1
update:	
3.586732e-06
======================================================================
                             Running SCRF
----------------------------------------------------------------------
Iter:	
1
update:	
1.734644e-06
    1        6.815819e-05         -7.321950145174       -9.038879e-07
----------------------------------------------------------------------
                   SCF converged in 1 iterations!
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                 Printing Molecular Properties                  ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            1
 Multiplicity          :                                            1
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    2    p        :           (au)               -2.247855037079
----------------------------------------------------------------------
 Sum occupied          :           (au)               -4.495710074159
======================================================================


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.221577368241
 E-N energy            :           (au)              -16.099470337524
 Coulomb energy        :           (au)                3.286431013726
 Exchange energy       :           (au)               -0.410794633751
 X-C energy            :           (au)               -1.259143305750
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
 Tot. Reac. Energy     :           (au)               -0.060550250115
 El. Reac. Energy      :           (au)                0.121121010921
 Nuc. Reac. Energy     :           (au)               -0.181671261035
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.140278884138
 Nuclear energy        :           (au)               -0.181671261035
----------------------------------------------------------------------
 Total energy          :           (au)           -7.321950145174e+00
                       :     (kcal/mol)           -4.594593047642e+03
                       :       (kJ/mol)           -1.922377732381e+04
                       :           (eV)           -1.992404047172e+02
======================================================================


                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                         Exiting MRChem                         ***
***                                                                ***
***                    Wall time :  0h  0m 11s                     ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
